application-flashmessages-test/application-flashmessages-test-pageobjects/src/main/java/org/xwiki/flashmessages/test/po/FlashEntryEditPage.java

```diff
@@ -49,19 +49,19 @@
     @FindBy(id = "Flash.FlashClass_0_repeat")
     private WebElement repeatElement;
 
-    @FindBy(xpath = "//label[@for='Flash.FlashClass_0_repeatInterval']")
+    @FindBy(xpath = "//div[@class='repeatSettings']/div[@class='row']/div[1]/dl/dt/label")
```

Comment:
It would be better to find for all these `xpath` expressions the `css`  correspondents selectors and try to write them as simple as possible.